### DISCUSSION TITLE
Right click context menu for dags and tasks in Grid view

### DISCUSSION THREAD

**Discussion by Adaverse** [apache/airflow#33628] [Ideas]
### Description

It would be really nice to have right-click context menu for dags and tasks in the grid view which would contain links to common actions like clear, mark fail, view logs etc.... like below -
 
![image](https://github.com/apache/airflow/assets/35839624/e8935b39-21e5-42b4-8a83-dea7915e9027)

### Use case/motivation

Simpler navigation when performing these actions across dags and tasks.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by phanikumv** [comment]
cc @bbovenzi any thoughts on this?

**Comment by hussein-awala** [comment]
+1 it would be a great feature!

**Comment by eladkal** [comment]
Isn't this covered by the key board shortcuts?

**Comment by bbovenzi** [comment]
> Isn't this covered by the key board shortcuts?

I would say so. but if someone wants to add this, they can go for it.

**Comment by eladkal** [comment]
Since this functionality already exist in a reasonable manner I'll convert it to discussions: ideas.
if anyone wants to implement the above suggestion feel free to open PR directly
